addons/toolbars/src/types.ts

```diff
@@ -31,6 +31,7 @@ export interface NormalizedToolbarConfig {
   preventDynamicIcon?: boolean;
   items: ToolbarItem[];
   shortcuts?: ToolbarShortcuts;
+  toggle?: boolean;
```

Comment:
Would a more descriptive prop be `cycle`? `isCycling` feels more accurate than `isToggle` to me. What do you think?

Reply:
> Would a more descriptive prop be `cycle`? `isCycling` feels more accurate than `isToggle` to me. What do you think?

It was cycle, then changed to toggle - I'll let others decide, I was worried about "cycle" not be too apparent in what it does, where "toggle" is easier to grasp what it does - but is technically not right